tests/expose.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('expose', () => {
     expect(wrapper.vm.other).toBe('other')
     // can't access `msg` as it is not exposed
     // and we are in a component with a setup returning a render function
-    expect(wrapper.vm.msg).toBeUndefined()
+    expect((wrapper.vm as unknown as { msg: undefined }).msg).toBeUndefined()
```

Comment:
Should we be ts-ignoring these or is there more work to come (maybe in Volar) before merging this?

Reply:
This one is expected to not compile without `as`, as the test is just to check it is _not_ exposed 😉 So I think it's fine to explicitly cast it to make TS happy.